packages/server/src/routes/calendar/calendar.test.ts

```
@@ -0,0 +1,98 @@
+					},
+				},
+				{
+					headers: {
+						authorization: `Bearer ${"bearetoken123"}`,
+					},
+				},
+			);
+
+			if (evenetResponse.status !== 201) {
+				const error = await evenetResponse.json();
+				throw new Error("Error creating event: " + error.error);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)